### ISSUE TITLE
Allow live stream when participant password/pin is set

### ISSUE THREAD

**Issue by fredposner** [jitsi/jibri#339]
*This Issue tracker is only for reporting bugs and tracking code related issues.*

[*] Before posting, please make sure you check community.jitsi.org to see if the same or similar bugs have already been discussed. General questions, installation help, and feature requests can also be posted to community.jitsi.org.

## Live streaming fails when participant password/pin is set ##

When limiting a conference to participants through use of password/pin, jibri is unable to join the conference to live stream.

## Current behavior ##

Live streaming fails. When password/PIN is removed, live streaming works without issue. Currently one would need to remove pin (like a grenade), stream, then re-add the pin when the streaming has started; opening the room to unwanted guests.

## Expected Behavior ##

Security pin/password passed to jibri to join conference to then stream.

## Possible Solution ##

A user field to pass the pin can be used if jibri can be allowed to override the PIN/pull from system.

## Steps to reproduce ##

1. Start conference
2. Add PIN/Password
3. Attempt Live Stream
4. Re-attempt live stream ;)
5. Remove PIN/Password
6. Live stream

# Environment details #

Docker build on debian 10.5

**Comment by fredposner** [comment]
Edit to add... long time user, big fan. Thank you for this great software and your contributions.

**Comment by damencho** [comment]
There are few posts around that in the community forum and a sample custom lua module you can add that can fix this.

**Comment by damencho** [comment]
Don't have the links handy, sorry.

**Comment by fredposner** [comment]
I didn't find any in a search, but would be interested in an approach or a built-in bypass.

**Comment by damencho** [comment]
https://community.jitsi.org/t/jibri-authenticating-if-a-room-has-a-password-added-by-moderator/18775

**Comment by fredposner** [comment]
Thank you for this. I think this is honestly outside the ability of most users; especially when deploying in docker. Would it be possible to transfer this issue to an enhancement / feature request perhaps vs bug then?

**Comment by damencho** [comment]
Yep, create an issue about it in jitsi-meet project as there are the prosody plugins. 
Any help is welcome, we will review any PR, but not sure when we will get to add it as we are hands full of work ...
